src/base/bittorrent/torrenthandle.cpp

```diff
@@ -428,7 +436,9 @@ QHash<QString, TrackerInfo> TorrentHandle::trackerInfos() const
 
 void TorrentHandle::addTrackers(const QVector<TrackerEntry> &trackers)
 {
-    const QVector<TrackerEntry> currentTrackers = this->trackers();
+    QSet<TrackerEntry> currentTrackers;
```

Comment:
There is `reserve()` in QSet too.

Reply:
https://doc.qt.io/qt-5/qset.html#reserve
>In general, you will rarely ever need to call this function.

Maybe we are safe not to add it?

Reply:
Seems Yes.